packit_service/worker/handlers/distgit.py

```diff
@@ -460,6 +495,17 @@ def pre_check(self) -> bool:
                         f"configuration: {self.job_config.allowed_committers}."
                     )
                     return False
+        elif self.data.event_type in (PullRequestCommentPagureEvent.__name__,):
+            commenter = self.data.actor
+            logger.debug(
+                f"Triggering downstream koji build through comment by: {commenter}"
+            )
+            if not self.is_packager(commenter):
```

Comment:
in this case, it could be nice to notify user about this, but we can discuss if and how we want to do it (PR comment?) as a follow-up and be also consistent with retriggering bodhi updates